go/common/sgx/ias/quote.go

```diff
@@ -146,6 +164,14 @@ func (q *Quote) Verify() error {
 	if mrSignerBlacklist[q.Report.MRSIGNER] {
 		return fmt.Errorf("ias/quote: blacklisted MRSIGNER")
 	}
+
+	// Drop enclaves compiled in debug mode.
```

Comment:
Random thought.  Should the unsafe flag, mandate debug enclaves?  IMO it should be either production enclaves OR debug enclaves, not "set a flag and be allowed to mix both".

Reply:
Hm.. but this will prevent us from doing any tests with the production enclave. Will this never be the case?

Reply:
Fixed.